src/components/style/collection.css

```diff
@@ -1,3 +1,4 @@
+/* Why not be consistent and use only .css or .css.js? - Harold */
```

Comment:
we are using the css.js to do some inline styling currently.  We can try to get this to work with classes instead.  For a large portion of the project we were not able to get .css files to import properly so that also contributed to getting us into the current configuration